modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -214,6 +216,55 @@ The `IF NOT EXISTS` and `OR REPLACE` parts of these commands cannot be used toge
 ====
 
 
+==== Create databases using seeding options
+
+Neo4j has built-in support for a seed from a mounted file system (file), FTP server, HTTP/HTTPS server, Amazon S3, Google Cloud Storage, and Azure Cloud Storage.
+To select a seed provider, you need to configure xref:configuration/configuration-settings.adoc#config_dbms.databases.seed_from_uri_providers[dbms.databases.seed_from_uri_providers].
```

Comment:
I see on the other page that this configuration is only for a mounted file system (file), FTP server, HTTP/HTTPS server.

Reply:
Maybe we need the text from the note here instead.

Reply:
Where is this page?

This config's default is `CloudSeedProvider` (for Amazon S3, Google Cloud Storage, and Azure Cloud Storage). If you want to use another, you need to configure this manually.

Reply:
> Where is this page?
> 
> This config's default is `CloudSeedProvider` (for Amazon S3, Google Cloud Storage, and Azure Cloud Storage). If you want to use another, you need to configure this manually.

Yes, this is correct. But from the sentence, I understand that I need to configure a seed provider regardless of what provider I want to use.